pkg/apis/go.mod

```diff
@@ -36,24 +36,24 @@ require (
 	github.com/rancher/gke-operator v1.9.3
 	github.com/rancher/norman v0.0.0-20240708202514-a0127673d1b9
 	github.com/rancher/rke v1.6.3
-	github.com/rancher/wrangler/v3 v3.0.0
+	github.com/rancher/wrangler/v3 v3.0.1-rc.2
```

Comment:
Note to reviewers:
We need to match the version wit the one in the root `go.mod` file to make the CI happy.